tests/conftest.py

```diff
@@ -116,6 +140,26 @@ def small_sky_order1_catalog(small_sky_order1_dir):
     return lsdb.read_hipscat(small_sky_order1_dir)
 
 
+@pytest.fixture
+def small_sky_order1_source_catalog(small_sky_order1_source_dir, small_sky_order1_source_margin_catalog):
```

Comment:
```suggestion
def small_sky_order1_source_with_margin(small_sky_order1_source_dir, small_sky_order1_source_margin_catalog):
```

I know this fixture already has a long name, but it would make the tests a lot clearer that this catalog already has a margin included.